### PR TITLE
Updating Dockerfile, removing unused files, and fixing regex pattern …

### PR DESCRIPTION
**Pull request by ewolinetz**
…in fluentd.conf

### CODE REVIEW

**Comment by ewolinetz** [comment]
This is to address issue where fluentd was receiving this message due to date format:
```
2016-01-06 20:53:09 +0000 [warn]: got incomplete line before first line from /var/log/messages: "Jan  6 15:53:09 ip-172-18-2-62 docker: time=\"2016-01-06T15:53:09.871067358-05:00\" level=info msg=\"GET /version\"\n"
```

**Comment by ewolinetz** [comment]
@sosiouxme  @jcantrill PTAL

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -23,7 +23,6 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
   mkdir -p ${HOME} && \
   gem install --no-ri fluentd fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch fluent-plugin-flatten-hash
 
-ADD out_elasticsearch*.rb /opt/app-root/src/gems/fluent-plugin-elasticsearch-1.0.0/lib/fluent/plugin/
```
Comment (sosiouxme) [inline fluentd/Dockerfile]: @ewolinetz I assume this is being added in a different way now?

Comment (ewolinetz) [inline fluentd/Dockerfile]: @sosiouxme yeah, the latest release of fluent-plugin-elasticsearch contains these changes now, we were already installing it in the line above but then overlaying the files we were providing

**Comment by sosiouxme** [comment]
LGTM

**Pull request merged**